node/src/command.rs

```diff
@@ -46,6 +46,8 @@ fn load_spec(
 		)?),
 		"statemine-dev" => Box::new(chain_spec::statemine_development_config(para_id)),
 		"statemine-local" => Box::new(chain_spec::statemine_local_config(para_id)),
+		"westmint-dev" => Box::new(chain_spec::westmint_development_config(para_id)),
+		"westmint-local" => Box::new(chain_spec::westmint_local_config(para_id)),
```

Comment:
what about `statemine` and `westmint` specs?

Reply:
Tracking here: https://github.com/paritytech/statemint/issues/14